### ISSUE TITLE
Thinkpad e14 gen2 supported?

### ISSUE THREAD

**Issue by monlor** [AniKulkarn/Hackintosh-ThinkPad-E14#29]

**Comment by AniKulkarn** [comment]
No. This EFI will not support any other laptop as it is. You will have to generate your own EFI based on the Dortania guide. However, you may use the README and the contents of the EFI as a reference for solving problems with your new EFI (like battery indication, sound, Wi-Fi, Sleep/Wake, etc.). DO NOT USE THIS EFI DIRECTLY ON THE 2nd GEN THINKPAD E14.
